### ISSUE TITLE
Request to add Smaug Llama3

### ISSUE THREAD

**Issue by rjmalagon** [ollama/ollama#4665]
This fine tune model of Llama-3-70B-Instruct target to be a very high performant on quality
https://huggingface.co/abacusai/Smaug-Llama-3-70B-Instruct

Similar to Smaug-72b (but improved) that was built and finetune with DPO techniques https://arxiv.org/abs/2402.13228
Summarized: "DPO-Positive (DPOP), a new loss function and training procedure which avoids reduction of the DPO model’s intended picking one response over another."

Recently added to llama.cpp
https://github.com/ggerganov/llama.cpp/commit/c429b33beb35f13934a4dfbe0c138d30b45e5d54

**Comment by rick-github** [comment]
```console
$ ollama run hf.co/mradermacher/Smaug-Llama-3-70B-Instruct-GGUF:Q4_K_M
pulling manifest 
pulling f5f93e3d8573: 100% ▕█████████████████████████████████████████████████████████████████▏  42 GB    
pulling 62fbfd9ed093: 100% ▕█████████████████████████████████████████████████████████████████▏  182 B
pulling b78301c0df4d: 100% ▕█████████████████████████████████████████████████████████████████▏   38 B
pulling b972d68f7676: 100% ▕█████████████████████████████████████████████████████████████████▏  552 B
verifying sha256 digest 
writing manifest 
success 
>>> hello
Hi! It's nice to meet you. Is there something I can help you with or would you like to chat?
```
